src/cocotb_tools/runner.py

```diff
@@ -491,6 +529,40 @@ def get_abs_paths(paths: Sequence[PathLike]) -> List[Path]:
     return [get_abs_path(path) for path in paths]
 
 
+_verilog_extensions = (".v", ".sv", ".vh", ".svh")
```

Comment:
Actually... are `.vh` and `.svh` not files that should only be included and not directly compiled? We might say that users shouldn't list them in the sources then, but it would allow them to just do a recursive glob over a toplevel directory without extra filtering.

Maybe we should expose the extensions as a user-definable option?

Reply:
I'm seen some stuff you wouldn't imagine... And from major vendors at that. Sometimes header files need to be compiled as source files and not just included.